pages/settings/privacy/index.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import Glean from '@mozilla/glean/web'
+import { mastodonAccountHandle, mastodonAccountId } from '~/telemetry/generated/identifiers'
+import { userAgent } from '~~/telemetry/generated/identifiers'
+
+const user = currentUser.value
+
+const { t } = useI18n()
+
+useHydratedHead({
+  title: () => `${t('settings.privacy.label')} | ${t('nav.settings')}`,
+})
+const userSettings = useUserSettings()
+
+function toggleOptOut() {
+  const allowGlean = togglePreferences('allowGlean')
+  Glean.setUploadEnabled(allowGlean)
+
+  // reset identifiers if user opts back in
+  if (user && allowGlean) {
+    mastodonAccountHandle.set(user.account.acct)
+    mastodonAccountId.set(user.account.id)
+    userAgent.set(navigator.userAgent)
+  }
```

Comment:
Needed to add this because these identifiers were not getting reset when the user would opt out and then opt back in again.